server/middlewares/csp.js

```diff
@@ -14,7 +14,6 @@ export default function csp() {
   return helmet.contentSecurityPolicy({
     directives: {
       defaultSrc: trusted.concat([
-        '*.optimizely.com',
```

Comment:
I believe it may need exist in both place as optimizely will also add css and fonts.